src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/config/SecurityUserContext.kt

```diff
@@ -0,0 +1,24 @@
+package uk.gov.justice.digital.hmpps.externalusersapi.config
```

Comment:
Just a personal preference:I do like the way that auth does this - but I also like the naming convention used in other services (e.g. incentives api just has a method getUsername rather than principal in AuthenticationFacade).
Happy to accept this but I prefer username.